dbt_coves/utils/airbyte_api.py

```diff
@@ -0,0 +1,114 @@
+            raise AirbyteApiCallerException(
+                f"Error initializing Airbyte API Caller: Missing configuration: {e}"
+            )
+
+        airbyte_api_list_component = airbyte_api_base_endpoint + "{component}/list"
+        self.airbyte_endpoint_list_connections = airbyte_api_list_component.format(
+            component="connections"
+        )
+        self.airbyte_endpoint_list_sources = airbyte_api_list_component.format(
+            component="sources"
+        )
+        self.airbyte_endpoint_list_destinations = airbyte_api_list_component.format(
+            component="destinations"
+        )
+
+        airbyte_endpoint_list_workspaces = airbyte_api_list_component.format(
+            component="workspaces"
+        )
+
+        airbyte_api_create_component = airbyte_api_base_endpoint + "{component}/create"
+        self.airbyte_endpoint_create_connections = airbyte_api_create_component.format(
+            component="connections"
+        )
+        self.airbyte_endpoint_create_sources = airbyte_api_create_component.format(
+            component="sources"
+        )
+        self.airbyte_endpoint_create_destinations = airbyte_api_create_component.format(
+            component="destinations"
+        )
+
+        airbyte_api_update_component = airbyte_api_base_endpoint + "{component}/update"
+        self.airbyte_endpoint_update_sources = airbyte_api_update_component.format(
+            component="sources"
+        )
+        self.airbyte_endpoint_update_destinations = airbyte_api_update_component.format(
+            component="destinations"
+        )
+        self.airbyte_endpoint_delete_connection = (
+            airbyte_api_base_endpoint + "connections/delete"
+        )
+
+        self.airbyte_endpoint_list_destination_definitions = (
+            airbyte_api_base_endpoint + "destination_definitions/list"
+        )
+
+        self.airbyte_endpoint_list_source_definitions = (
+            airbyte_api_base_endpoint + "source_definitions/list"
+        )
+
+        try:
+            self.airbyte_workspace_id = self.api_call(airbyte_endpoint_list_workspaces)[
+                "workspaces"
+            ][0]["workspaceId"]
+            self.standard_request_body = {"workspaceId": self.airbyte_workspace_id}
+            self.airbyte_connections_list = self.api_call(
+                self.airbyte_endpoint_list_connections, self.standard_request_body
+            )["connections"]
+            self.airbyte_sources_list = self.api_call(
+                self.airbyte_endpoint_list_sources, self.standard_request_body
+            )["sources"]
+            self.airbyte_destinations_list = self.api_call(
+                self.airbyte_endpoint_list_destinations, self.standard_request_body
+            )["destinations"]
+        except:
+            raise AirbyteApiCallerException(
+                "Couldn't retrieve Airbyte connections, sources and destinations"
+            )
+
+    def load_definitions(self):
```

Comment:
Who calls this function?

Reply:
`load_definitions()` is called by `dbt-coves load` to make source and destination definitions available for their creations. The                                                                      doubt I had was whether returning those lists to "live" in `dbt-coves load` or keeping them in `airbyte-api-caller` (it's currently in this last one)

Reply:
Can´t find the file/line where it´s called.

Reply:
Inside load/airbyte.py, line 82

![image](https://user-images.githubusercontent.com/22618882/144236240-244dfa5f-fa19-4e08-863b-93b80084f7ec.png)


I did it that way because extract doesn't need those definitions and, therefore, loading them in api_caller's instantiation seemed a waste of resources